lib/plugins/aws/package/index.js

```diff
@@ -55,6 +56,20 @@ class AwsPackage {
     };
 
     this.hooks = {
+      'initialize': () => {
+        const isPackageCommand =
+          this.serverless.processedInput.commands.length === 1 &&
+          this.serverless.processedInput.commands[0] === 'package';
```

Comment:
Shorter form could be: `this.serverless.processedInput.commands.join(" ") === "package"`